backend/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM php:7.4-fpm-alpine
```

Comment:
nit: Do we need `fpm` if we run artisan?
```suggestion
FROM php:7.4-alpine
```

Reply:
I will make a test to see if it's working without fpm

Reply:
You were right! I made another change, Thank you!